src/components/Timeline.js

```diff
@@ -2,9 +2,21 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
-  
-  return;
+const Timeline = (props) => {
+      const timeline = props.events.map((event) => {
+        return(
+          <TimelineEvent 
+            person={event.person}
+            status={event.status}
+            timeStamp={event.timeStamp} />
+        )
+      })
```

Comment:
You should add a key attribute when you return an array of components. 
```suggestion
      const timeline = props.events.map((event, index) => {
        return(
          <TimelineEvent 
            person={event.person}
            key={index}
            status={event.status}
            timeStamp={event.timeStamp} />
        )
      });
```